src/app/modules/account/components/update-request/update-request.component.html

```diff
@@ -110,17 +110,45 @@ <h2><strong>Documents - Update name due to name change</strong></h2>
     </common-checkbox>
 </div>
 
-<div *ngIf="person.updateGender" class="add-document">
+<div *ngIf="person.updateGender" class="add-document mb-4">
     <msp-support-documents
         [supportDocList]="genderChangeDocs"
         [(supportDoc)]="person.updateGenderDocType">
         <div sectionTitleInfo>
             <h2><strong>Documents - Correct gender designation - due to change</strong></h2>
-            <p class="border-bottom">
+            <p class="horizontal-line">
                 Provide the required document(s) to support your personal information update.
             </p>
+            <!-- <common-page-section layout="tips">
```

Comment:
Commented code.

Reply:
Ohh that one was a code that Im trying to make it work. I'll remove that code and update my PR. Thanks Sam.